vendor/bat-native-ads/src/bat/ads/internal/base/unittest/unittest_base.cc

```diff
@@ -65,6 +65,10 @@ AdsImpl* UnitTestBase::GetAds() const {
   return ads_.get();
 }
 
+Database* UnitTestBase::GetDatabase() const {
+  return database_.get();
+}
+
```

Comment:
We should not expose `Database`. Could we not call the Getters from the `_database.h` files for this use case?

Reply:
Refactored code not to use Database